src/Facebook/FacebookBatchRequest.php

```diff
@@ -91,6 +92,10 @@ public function add($request, $name = null)
             'request' => $request,
         ];
 
+        if (!empty($options)) {
+            $requestToAdd['options'] = $options;
```

Comment:
I'd move this in the array initialization above, no need to check for empty, it will always be an array thanks to the typehint